### DISCUSSION TITLE
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR message

### DISCUSSION THREAD

**Discussion by BunkerBuster73** [danielmiessler/Fabric#754] [Q&A]
I'm on a MBPro M3 pro and every time i type a fabric command i receive this message:

WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
I0000 00:00:1721574295.280747 1025158 config.cc:230] gRPC experiments enabled: call_status_override_on_cancellation, event_engine_dns, event_engine_listener, http2_stats_fix, monitoring_experiment, pick_first_new, trace_record_callops, work_serializer_clears_time_cache
I0000 00:00:1721574295.290770 1025158 check_gcp_environment_no_op.cc:29] ALTS: Platforms other than Linux and Windows are not supported

This is not affecting results but is quite annoing. I'm a newbie, can you help me to solve this ?
thank you
Bunker

**Comment by stuboo** [comment]
1. Add the following to line 8 of ./installer/client/cli/fabric.py `os.environ["GRPC_VERBOSITY"] = "NONE"`
2. [Update fabric](https://github.com/danielmiessler/fabric?tab=readme-ov-file#updating)


Example:
![CleanShot 2024-07-22 at 19 10 11@2x](https://github.com/user-attachments/assets/f62d22d2-e7b3-4565-bcf4-71e8645fb1b3)

**Reply by BunkerBuster73** [reply]
thank you stuboo, this removed the "warning" (verbose) message!

**Comment by fearby** [comment]
Tried this and still get the same message

**Reply by BunkerBuster73** [reply]
the update after inserting the line code terminated correctly ?
for me helped to remove the warning message
